monkey/agent_plugins/exploiters/snmp/src/plugin.py

```diff
@@ -42,7 +44,7 @@ def __init__(
         otp_provider: IAgentOTPProvider,
         **kwargs,
     ):
-        exploit_client = StubSNMPExploitClient(agent_id, agent_event_publisher)
+        exploit_client = SNMPExploitClient(agent_id, agent_event_publisher, plugin_name)
```

Comment:
Doesn't SNMPExploitClient require SNMPClient?

Reply:
It has a default argument. I'll just remove the default argument and inject it. We'll want to use the SNMPClient anyways to implement should_attempt_exploit()